File: C++/BubbleSort.cpp
#include<iostream>
using namespace std;

bubbleSort(int Q[],int n){
  int i,j,temp;
  for(i=0;i<n;i++){
    for(j=0;j<n-i;j++){
      if(Q[j]>Q[j+1]){
        temp=Q[j];
        Q[j]=Q[j+1];
        Q[j+1]=temp;
      }
    }
  }
  cout<<"After sorting\n";
  for(i=0;i<n;i++){
    cout<<Q[i]<<" ";
  }
}

int main(){
  int n;
  cout<<"Enter size of array"<<endl;
  cin>>n;
  int Q[n],i;
  cout<<"Enter array element\n";
  for(i=0;i<n;i++){
    cin>>Q[i];
  }
  bubbleSort(Q,n);
}

File: Java/BubbleSort.java
import java.util.Scanner;

public class BubbleSort {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        System.out.println("Enter array size");
        int arySize = scn.nextInt();
        int[] ary = new int[arySize];

        System.out.println("Enter array elements");
        for (int i = 0; i < arySize; i++) {
            ary[i] = scn.nextInt();
        }
        sub(ary);
    }

    public static void sub(int[] ary) {
        for (int i = 0; i < ary.length; i++) {
            for (int j = 0; j < ary.length - i - 1; j++) {
                if (ary[j] > ary[j + 1]) {
                    int temp = ary[j + 1];
                    ary[j + 1] = ary[j];
                    ary[j] = temp;
                }
            }
        }
        System.out.println("Sorted array");
        for (int k = 0; k < ary.length; k++) {
            System.out.print(ary[k] + " ");
        }
    }

}